cmd/tunneld/tunneld.go

```diff
@@ -138,3 +150,87 @@ func fatal(format string, a ...interface{}) {
 	fmt.Fprint(os.Stderr, "\n")
 	os.Exit(1)
 }
+
+func startAutocert(opts *options, server *tunnel.Server, logger log.Logger) {
+	cacheDir, err := cacheDir(opts.letsEncryptCacheDir)
```

Comment:
We must ensure that the directory exists, if it does not exist we must try to create it.

Reply:
This is handled by cert manager
https://github.com/jhernandezb/go-http-tunnel/blob/8d96dcb84113533659a3b841984019a735a36c01/vendor/golang.org/x/crypto/acme/autocert/cache.go#L67

Reply:
:+1: